release-notes/v0.6.0.md

```diff
@@ -0,0 +1,153 @@
+# What's new in Visual Studio Code Java?
+
+*February 2019*
+
+Welcome to the February 2019 release of Java Extension Pack. This document aims to recap the features, bug fixes around Java language support in between releases. Here are some of the key highlights:
+
+- **[IntelliCode](#intellicode)** for faster coding assisted by AI
+- **[Standalone Java Files](#standalone-java-files-supported)** better supported
+- **[Performance](#performance)** improved
+- **[Dependency Auto-Completion](#dependency-auto-completion-for-pom.xml-files)** for editing `pom.xml` files
```

Comment:
Broken link here

Reply:
In VS Code, the link works fine. I think VS Code uses a different markdown engine. This release note is optimized for VS Code audience. So I'll leave it as is.